nengo_gui/static/side_menu.css

```diff
@@ -0,0 +1,128 @@
+
+.sidenav-container {
+    height: calc(100% - 120px);
```

Comment:
I'm guessing this is supported in modern Chrome / Firefox / Safari which is what we're targeting right? Might be useful to explicitly state our minimum browser requirements so we can use things like these without worry.